src/shared/index.tsx

```diff
@@ -286,3 +286,79 @@ ReactDOM.render(
   </>,
   document.getElementById("experiment-4")
 );
+
+const experiment5 = {
+  "version": "1.0.0",
+  "metadata": {
+    "uuid": "e431af00-5ef9-44f8-a887-c76caa6ddde1",
+    "name": "Data Table Example",
+    "initials": "DT"
+  },
+  "schema": {
+    "sections": [
+      {
+        "title": "Collect",
+        "icon": "collect",
+        "formFields": ["experimentData"]
+      }
+    ],
+    "dataSchema": {
+      "type": "object",
+      "required": [],
+      "properties": {
+        "experimentData": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": ["location"],
+            "properties": {
+              "location": {
+                "title": "Location",
+                "type": "string",
+                "readOnly": true
+              },
+              "noValidation": {
+                "title": "Integer (-100 to 100)",
+                "type": "integer",
+                "minimum": -100,
+                "maximum": 100
+              },
+              "validation": {
+                "title": "Float (-100 to 100)",
+                "type": "number",
+                "minimum": -100,
+                "maximum": 100
+              }
```

Comment:
My question here is:  What causes one of these items to use validation, and one not to? Is it the label `noValidation` or ?

Reply:
I changed it when I added integer validation and forgot to update the label. :)